### ISSUE TITLE
New APIs for messages

### ISSUE THREAD

**Issue by 3RLikeNoOther** [abraham/twitteroauth#651]
Hi,
as you may know Twitter will change APIs for managing messages. 
At the moment I'm trying the one to send a message:
https://developer.twitter.com/en/docs/direct-messages/sending-and-receiving/api-reference/new-event
Using the code below I get null as result. 
Is my code wrong or the library doesn't support these new APIs?
Thanks

```
$connection = new TwitterOAuth( ... );

$parameters = array(
    'event' => array(
        'type' => "message_create",
        'message_create' => array(
            'target' => array(
                'recipient_id' => 306993015
            ),
            'message_data' => array(
                'text' => "Hello world"
            )
        )
    )
);

$result = $connection->post("direct_messages/events/new", $parameters);

echo json_encode($result,JSON_PRETTY_PRINT);
```

**Comment by Langsdorf** [comment]
Thank you!! @pklosko

**Comment by JosephShenton** [comment]
Hi @pklosko do you mind creating a fork with your changes please? I do not understand where these changes go.

**Comment by Langsdorf** [comment]
@JosephShenton  see TwitterOAuth.php

**Comment by JosephShenton** [comment]
Yes, but where in it do I put that code and what do I replace

**Comment by pklosko** [comment]
@JosephShenton 
Sorry. I'm out of PC.
OK I'll do it.
Probably during next week

**Comment by derikb** [comment]
Would love it if a fix for this could get into a release. That Twitter DM API change is coming up in August and I'd hate to have to try some workaround.

**Comment by abraham** [comment]
This is now available in v0.9
